juju/juju
# [JUJU-1369] Make juju-wait-for a standard command instead of a plugin

Moves `juju wait-for` from being a plugin to being a standard Juju command.

We also remove the make jobs and snapcraft logic for the plugin. We will need to update the release jobs to eschew packing what was previously a stand-alone plugin for the command.

## QA steps

- Bootstrap and add a model.
- `juju help wait-for`.
- `juju help wait-for application`.
- `juju wait-for application mysql --query='name=="mysql" && (status=="active" || status=="idle")'`
- In another terminal: `juju deploy mysql --series xenial`
- See the `wait-for` call return once the deployment quiesces.

## Documentation changes

There are probably references to it.

## Bug reference

N/A
